src/main/resources/feedbackQuestionMcqResultStatsTemplate.html

```diff
@@ -7,25 +7,32 @@
     </div>
   </div>
   <div class="row">
-    <div class="col-sm-12">
-      <table class="table margin-0">
+    <div class="col-sm-12 table-responsive">
+      <table class="table table-bordered table-striped margin-0">
```

Comment:
>Can this table

Yes, I can make them sortable.

>should this table be sortable?

IMHO I think even if we do that, it will not be used a lot. 
@xpdavid @bqnguyen94 @damithc What do you think? Should we do it?

Reply:
If there are many options and instructors want to sort them based on percentage or average. This might be helpful.

Reply:
Yes, we can make it sortable if it's not too much extra work.

Reply:
Okay. Will make them sortable.